plopfile.mjs

```diff
@@ -0,0 +1,88 @@
+import fs from "fs";
```

Comment:
Can we move this out of the root folder and into one of the helper/scripts directories